doc/orchestrator.rst

```diff
@@ -165,3 +165,287 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+object and assign it to the 'exp' variable. In this example, we instantiate the
+Experiment object with the name 'getting-started-multidb' and the 'launcher=slurm'
+configuration.
+
+.. code-block:: python
+  import numpy as np
+  from smartredis import Client
+  from smartsim import Experiment
+
+  exp = Experiment("getting-started-multidb", launcher="slurm")
+
+
+In the context of this Experiment, it's essential to create and launch
+the databases as a preliminary step before any other components. We aim
+to showcase the multi-database automation capabilities of SmartSim, so we
+create two types of databases in the workflow: a standalone database and a
+clustered database.
+
+To create a database, you can utilize the create_database() function located
+in the Experiment class. For launching multiple databases, this function requires
+specifying a unique database identifier argument named db_identifier. The db_identifier
+argument plays a crucial role in SmartSim multi-database support by serving as a unique
+identifier for each database being launched. The db_identifier will be utilized later
+in the example.
+
+Here's how to create the databases:
+
+For the standalone database:
+
+.. code-block:: python
+  standalone_database = exp.create_database(port=6379, db_nodes=1, interface="ib0", db_identifier="std-deployment")
+  exp.generate(standalone_database, overwrite=True)
+
+For the clustered database:
+
+.. code-block:: python
+  clustered_database = exp.create_database(port=6380, db_nodes=3, interface="ib0", db_identifier="clus-deployment")
+  exp.generate(clustered_database, overwrite=True)
+
+Next, pass the database instances to the following command:
+
+.. code-block:: python
+  exp.start(standalone_database, clustered_database, block=False, summary=True)
+
+The Experiment.start() function launches the Orchestrators for use within the workflow.
+
+.. note::
+
+  The Orchestrator is an in-memory database that can be launched alongside entities in SmartSim
+
+Next in the example, learn to distribute data between the launched databases by using the Client SmartRedis API.
+
+The SmartRedis Client object contains functions that manipulate, send, and receive
+data within the database. Each database has a single, dedicated SmartRedis Client.
+Begin by initializing a SmartRedis Client object per launched database.
+
+To create a designated SmartRedis Client, you need to specify the address of the target
+running database. You can easily retrieve this address using the get_address() function
+provided by the initialized Orchestrator object.
+
+.. note::
+
+  Make sure to set the 'cluster' argument appropriately based on whether the database is in a clustered configuration or not.
+
+Here's the code example:
+
+.. code-block:: python
+  client1 = Client(address=standalone_database.get_address()[0], cluster=False)
+  client2 = Client(address=clustered_database.get_address()[0], cluster=True)
```

Comment:
Please reverse the order of these parameters to cluster, then address. Even though they're kwargs and the order doesn't matter, it's better to match the order in the API